src/Compilers/CSharp/Portable/Lowering/LambdaRewriter/LambdaRewriter.Analysis.cs

```diff
@@ -73,7 +73,15 @@ internal sealed class Analysis : BoundTreeWalkerWithStackGuardWithoutRecursionOn
             public readonly HashSet<MethodSymbol> MethodsConvertedToDelegates = new HashSet<MethodSymbol>();
 
             /// <summary>
-            /// Any scope that a method in <see cref="MethodsConvertedToDelegates"/> closes over. If a scope is in this set, don't use a struct closure.
+            /// True if the method signature can't be rewritten to contain ref/out parameters.
+            /// </summary>
+            public bool CantTakeRefParameters(MethodSymbol closure) => closure.IsAsync
```

Comment:
>CantTakeRefParameters [](start = 24, length = 21)

Minor: It is probably better to remove negation form the name ("CanTakeRefParameters") and adjust the returned value accordingly.